pkg/transaction/eventgenerator.go

```diff
@@ -117,6 +118,11 @@ func (e *Generator) createEvent(logEvent LogEvent, eventTime time.Time, metaData
 func (e *Generator) CreateEvents(summaryEvent LogEvent, detailEvents []LogEvent, eventTime time.Time, metaData common.MapStr, eventFields common.MapStr, privateData interface{}) ([]beat.Event, error) {
 	events := make([]beat.Event, 0)
 
+	// See if the uri is in the api exceptions list
+	if len(detailEvents) > 0 && e.isInAPIExceptionsList(detailEvents) {
```

Comment:
I would put the len check in the isInAPI function. This way anyone who ever might try to call the func doesn't have to always check the len also. The func should not nil ptr out if it gets passed an empty array.

Reply:
i had it like that originally... i guess i can put it back.
No one should be really calling this except the func its in now.  But I get it.